frontend/src/metabase/search/utils/user-search-params/user-search-params.ts

```diff
@@ -0,0 +1,21 @@
+import type { UserId } from "metabase-types/api";
+import type { SearchQueryParamValue } from "metabase/search/types";
+
+export const parseUserIdString = (
```

Comment:
In programming context the word ["parse" implies the string type](https://en.wikipedia.org/wiki/Parsing) (so it's a pleonasm):

```suggestion
export const parseUserId = (
```